docs/sources/installation/debian.md

```diff
@@ -153,7 +153,7 @@ Start Grafana by running:
 
 ## Next steps
 
-Refer to the [Getting Started](/guides/getting_started/) guide for information about logging in, setting up data sources, and so on.
+Refer to the [Getting Started]({{< relref "getting_started.md" >}}) guide for information about logging in, setting up data sources, and so on.
```

Comment:
This maybe work, but looked at rpm installation and there the following relref is used:
```
instructions [here]({{< relref "../guides/getting_started/" >}})
```

Not sure which one is preferred, but would be good if we can be consistent. Maybe it doesn't care?

Reply:
It doesn't care. `relref` looks for the file name. We only need to specify the folder or part of the path if there is more than one identical file name. When I rewrite the RPM installation instructions, I will take a look.